.github/workflows/pr.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  Terraform:
+    name: Terraform Plan
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+
+    - name: Terraform Setup
+      uses: hashicorp/setup-terraform@v1
+
+    - name: Setup Terraform Cloud Credentials
+      run: echo "credentials \"app.terraform.io\" { token = \"${TFE_TOKEN}\" }" > ~/.terraformrc
+      env:
+        TFE_TOKEN: ${{ secrets.TFE_TOKEN }}
```

Comment:
You don't have access to secrets in a PR. It'd make it easy to print a secret and there's no way to securely use secrets on untrusted code.

Reply:
You do when the PR is not from another fork. (This wouldn't otherwise work)